x-pack/plugins/reporting/public/management/report_info_button.tsx

```diff
@@ -15,30 +15,33 @@ import {
   EuiSpacer,
   EuiText,
   EuiTitle,
+  EuiToolTip,
 } from '@elastic/eui';
 import { get } from 'lodash';
+import { injectI18n } from '@kbn/i18n/react';
 import React, { Component, Fragment } from 'react';
 import { USES_HEADLESS_JOB_TYPES } from '../../common/constants';
-import { ReportApiJSON } from '../../common/types';
+import { Job } from '../lib/job';
 import { ReportingAPIClient } from '../lib/reporting_api_client';
+import { Props as ListingProps } from './report_listing';
 
-interface Props {
+type Props = {
   jobId: string;
   apiClient: ReportingAPIClient;
-}
+} & Pick<ListingProps, 'apiClient' | 'intl'>;
```

Comment:
This component is updated with a minor UI improvement to show a tooltip over the "info" icon. This makes it consistent with the "download" icon.